perun-core/src/main/java/cz/metacentrum/perun/core/implApi/MembersManagerImplApi.java

```diff
@@ -457,4 +457,12 @@
 	 * @throws NamespaceRulesNotExistsException of there are no rules for the namespace
 	 */
 	NamespaceRules getNamespaceRules(String namespace) throws NamespaceRulesNotExistsException;
+
+	/**
+	 * Reject all member applications which are not Approved or already rejected
+	 *
+	 * @param sess perunSession
+	 * @param member which applications will be removed
+	 */
+	void rejectAllMemberOpenApplications(PerunSession sess, Member member);
```

Comment:
It might be worth mentioning that no notifications are sent when using this method.

Reply:
@balcirakpeter Hi, could you please take a look at this so we can merge this PR?